### PR TITLE
Add "Latest news broadcast"

### PR DESCRIPTION
**Pull request by rysport**
Adds "latest news" as a menu option. Gets content from www.svtplay.se/nyheter. Live broadcasts work.
image

### CODE REVIEW

**Comment by linqcan** [comment]
Thanks! Will look into this today. The nice weather has kept me afk.

File: resources/lib/svt.py [hunk 1]
```diff
@@ -24,6 +25,12 @@
 SEARCH_LIST_EPISODES = "[^\"']*playJs-search-episodes[^\"']*"
 SEARCH_LIST_CLIPS = "[^\"']*playJs-search-clips[^\"']*"
 
+def getLatestNews():
+  """
+  Returns a list of latest news.
+  """
+  sliderId = "playJs-latest"
```
Comment (linqcan) [inline resources/lib/svt.py:32]: No camel casing on variables is the convention, and here I don't see the purpose of having a variable either. Scrap it and put string arg directly into the function call instead.

**Comment by linqcan** [comment]
I messed up, change according to my comments and make a new PR and I will merge it.

**Comment by rysport** [comment]
Sure :) I will send a new PR soon

**Pull request merged**